README.md

```diff
@@ -115,8 +115,8 @@ tasks.dokkaHtml.configure {
 
 Groovy
 ```kotlin
```

Comment:
We might also want to fix this here also, right? 🙄

Reply:
I didn't test it so I have no idea how it will work out in Kotlin DSL
I could test it on Kotlin DSL as well if you want me to do so

Reply:
Oh I think we can just replace ```kotlin with ```groovy ☺️

Reply:
I think you don't need to specify a language there?

Reply:
I think you should replace `kotlin` with `gradle`  
  
source:  
[Github official guide](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting)
[Supported Language List](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1891)